qiskit/providers/ibmq/api_v2/clients/account.py

```diff
@@ -16,20 +16,32 @@
 
 import asyncio
 
+from typing import List, Dict, Any
+# Disabled unused-import because datetime is used only for type hints.
+from datetime import datetime  # pylint: disable=unused-import
+
 from ..rest import Api
 from ..session import RetrySession
 
 from .base import BaseClient
 from .websocket import WebsocketClient
 
+# Type aliases for type hints.
```

Comment:
This is probably a leftover from a previous iteration?

Reply:
Yeah, this was leftover. Thanks for the catch! I'll make sure to remove it 👍🏽.